### ISSUE TITLE
Jobs page: Add the link to the Discourse forum post of rules for responding to a job and add the rules

### ISSUE THREAD

**Issue by Erioldoesdesign** [opensourcedesign/opensourcedesign.github.io#459]
We've had a higher rate of what seem to be 'copy-paste' responses to jobs posts this year - one way we're limiting that is introducing a rule that if you're posting a reply in our forum to a job opening from our jobs board that you avoid copy-paste like responses and you will get 3 warnings before being banned.

You can find info here: https://discourse.opensourcedesign.net/t/guidelines-on-posting-and-responding-to-jobs/3416

I think that another location to surface this information is the [jobs page](https://opensourcedesign.net/jobs/) itself! 

Let's link to this forum pots in the top of the jobs page and also consider having text of the rules in order to try to avoid copy-paste responses from designers.

**Comment by jdittrich** [comment]
> I think that another location to surface this information is the [jobs page](https://opensourcedesign.net/jobs/) itself!

Since there is no way to directly comment on the job page, I would probably not add the text here, since it diverts attention from things people can actually do there right now (posting jobs, browsing jobs)

**Comment by Erioldoesdesign** [comment]
> > I think that another location to surface this information is the [jobs page](https://opensourcedesign.net/jobs/) itself!
> 
> Since there is no way to directly comment on the job page, I would probably not add the text here, since it diverts attention from things people can actually do there right now (posting jobs, browsing jobs)

I think my justification was that a lot of designers wanting to respond to a job do it via the individual jobs page and then click into the specific discussion for that job, bypassing the notices in the discourse forum. 

My other suggestions for this would be adding it to the bottom of ever job posting right before the embedded forum discussion link and/or at the top of every created forum discussion individual job post. Thoughts?

**Comment by jdittrich** [comment]
> My other suggestions for this would be adding it to the bottom of ever job posting right before the embedded forum discussion link…

Yes, before or after the link would be a good place. A short text like "Don’t use generic copy-pasted responses; they can get you banned (See Guidelines)" would not take much space.

**Comment by Erioldoesdesign** [comment]
After some (too long) investigation in the job repo I think the perma link to the 'reply to job post' info needs to be added just above where this .html talks about discourse stuff:
https://github.com/opensourcedesign/jobs/blob/master/_layouts/jobs.html

**Comment by amiii123malviya** [comment]
Hello,

Is this issue still active?
I would like to work on this.

Best Regards.

**Comment by Erioldoesdesign** [comment]
> Hello,
> 
> Is this issue still active? I would like to work on this.
> 
> Best Regards.

I'll assign you this issue for 1 month and then un-assign if there's no progress - thank you 😄

**Comment by SenKVarghese** [comment]
Hey. I would like to work on this..
Best Regards

**Comment by DhawalShankar** [comment]
I would like to work on this. Please assign me the task!
